.github/workflows/reindex-website-main.yml

```diff
@@ -15,3 +15,27 @@ jobs:
           username: ${{ secrets.ALGOLIA_CRAWLER_USER_ID }}
           password: ${{ secrets.ALGOLIA_CRAWLER_API_KEY }}
           customHeaders: '{"Content-Type": "application/json"}'
+  check-index:
+    runs-on: ubuntu-latest
+    name: Checks that the latest index is still working
+    steps:
+      - uses: fjogeleit/http-request-action@v1
+        id: search
+        with:
+          url: "https://NTSX40VZB8-dsn.algolia.net/1/indexes/kurtosis/query"
+          method: 'POST'
+          customHeaders: '{"X-Algolia-API-Key": "4269c726c2fea4e6cddfeb9a21cd3d4e", "X-Algolia-Application-Id": "NTSX40VZB8"}'
```

Comment:
Not super familiar with Algolia API, but is this API key and app ID safe to share in our public monorepo?

Reply:
Yep you can see these on docs.kurtosis.com